local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -106,14 +106,17 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 
 // GetFsAvailableByte returns available capacity in byte about a mounted filesystem.
 // fullPath is the pathname of any file within the mounted filesystem. Capacity
-// returned here is total capacity available to non-root users, and does not include
-// fs reserved space.
+// returned here is total capacity available.
 func (u *volumeUtil) GetFsAvailableByte(fullPath string) (uint64, error) {
-	var s unix.Statfs_t
-	if err := unix.Statfs(fullPath, &s); err != nil {
+	//largely copied from kubernetes/pkg/volume/util/fs.go:FsInfo(path)
```

Comment:
There will only be issues if we use functions from kubernetes/kubernetes/pkg/volume/util/ that accept or return kubernetes/kubernetes/pkg/api types, because we want to work with downstream k8s.io/api types instead.

It seems Fsinfo doesn't have this issue, so we won't run into any compilation errors when vendoring kubernetes/kubernetes/pkg/volume/util/ & using it. But generally we want to solve this issue for long-term so we can make use of existing code https://github.com/kubernetes-incubator/external-storage/issues/165